app/gui/src/dashboard/pages/dashboard/Dashboard.tsx

```diff
@@ -104,14 +103,11 @@ function DashboardInner(props: DashboardProps) {
   const initialProjectName = initialLocalProjectPath != null ? null : initialProjectNameRaw
 
   const categoriesAPI = useCategoriesAPI()
-
```

Comment:
Where did we move the refetch directories though?

Reply:
we no longer batch refetch directories since there is no longer a tree, no? so only one directory ever needs to be refetched at any time

Reply:
Hm, that makes sense, but we need to be sure we added the refetch in options object, but I have no idea how to avoid turning the listDirectory options constructor into a hook (as we still rely on feature-flags here). So if you have an idea how to tackle this, I'm happy to delete these lines.

Reply:
i think my preferred solution would be to add a hook to return the required variables (enable refetch and refetch interval), and add an extra parameter to the constructor (*required* so that the user must explicitly opt out) that takes in the return type of that hook. thoughts?

Reply:
In that case it's easy to forget to add this, the same query folks use in the Editor IIRC, and I think it makes sense to avoid spreading logic across the app

Reply:
alternatively we can just have a hook `useListDirectory` maybe

Reply:
re: easy to forget, there's a mandatory new property to the listDirectory options constructor so at the very least it'd be noticeable if someone intentionally avoids passing it